cli/azd/pkg/infra/azure_resource_types.go

```diff
@@ -58,6 +62,14 @@ func GetResourceTypeDisplayName(resourceType AzureResourceType) string {
 		return "App Service plan"
 	case AzureResourceTypeCosmosDb:
 		return "Azure Cosmos DB"
+	case AzureResourceTypeApim:
+		return "Azure API Management"
+	case AzureResourceTypeCacheForRedis:
+		return "Cache for Redis"
+	case AzureResourceTypeCosmosMongo:
+		return "Azure Cosmos MongoDB"
+	case AzureResourceTypeCosmosSQL:
+		return "Azure Cosmos SQL DB"
```

Comment:
Can we also add regular SQL DB?

Reply:
#857 is referring to Standard SQL Server / DB, not Cosmos SQL.  I'm not sure we "need" to show the difference between different flavors of cosmos.

Reply:
Thank you @wbreza .

The issue here was to only add a missing case to the switch.  I have created this other PR: https://github.com/Azure/azure-dev/issues/857